src/spyglass/common/common_subject.py

```diff
@@ -17,51 +17,32 @@ class Subject(SpyglassMixin, dj.Manual):
     sex = "U": enum("M", "F", "U")
     species = NULL: varchar(200)
     """
+    _expected_duplicates = True
+
+    @property
+    def table_key_to_obj_attr(self):
+        return {
+            "self": {
+                "subject_id": "subject_id",
+                "age": "age",
+                "description": "description",
+                "genotype": "genotype",
+                "species": self.standardized_sex_string,
```

Comment:
This is smart accepting callables here - it lets the processing pieces get passed up the inheritance